pkg/watcher/watcher.go

```diff
@@ -29,23 +29,22 @@ func (w *Watcher) RunSync() error {
 	if w.Viper.GetString("driver") == "postgres" {
 		c, err := postgres.Connect(w.Viper.GetString("uri"))
 		if err != nil {
-			return err
+			return errors.Wrap(err, "postgres connect")
 		}
 
 		conn = c
 	} else if w.Viper.GetString("driver") == "mysql" {
 		c, err := mysql.Connect(w.Viper.GetString("uri"))
 		if err != nil {
-			return err
+			return errors.Wrap(err, "mysql connect")
```

Comment:
@emosbaugh we've settled on using verbs in the wrapped errors and then unwrapping them with "cause" in the outer loop. It's giving us very clean output that's easy to debug and consistent.  Would you mind updating these to a slightly more verbose format, such as:  `return errors.Wrap(err, "failed to connect to mysql")`?

We definitely want to avoid the wrapped output of "failed to... failed to ... failed to..." and this isn't consistent in this project yet, but the pattern has been working pretty well in others. Generally, we need to update this project with error wrapping all over (it's super rough right now), but as we drop a few in, it would be good to get the consistent format of these.

Reply:
I don’t understand. If you unwrap the error with cause why wrap?

Reply:
The issue I have without wrapping here is that it was unclear what was producing the error. Without wrapping I was getting the following. If I unwrap with cause before I log how will I know what is producing the error?

```
Error: pq: error in argument for $1: unsupported OID 2205 with format code FormatText
```